src/gui/mzroll/background_peaks_update.cpp

```diff
@@ -186,53 +186,8 @@ void BackgroundPeakUpdate::loadSettings(QString fileName) {
             if (xml.name() == "PeakDetectionSettings") {
                 settings->setValue("alignSamplesFlag", xml.attributes().value("alignSamplesFlag").toString().toInt());
```

Comment:
Why hasn't this been removed?

Reply:
I was focusing only  on removing peaks and options dialog settings.
Let's do this when doing the same with alignmentDialog ?

Reply:
Makes sense.